jenkins/agent-base/Dockerfile.rhel7

```diff
@@ -70,6 +71,16 @@ RUN if [ -z $SNYK_DISTRIBUTION_URL ] ; then echo 'Skipping snyk installation!' ;
     && echo 'Snyk installation completed!'; \
     fi
 
+# Optionally install Aquasec.
+RUN if [ -z $AQUA_SCANNERCLI_URL ] ; then echo 'Skipping AquaSec installation!' ; else echo 'Installing AquaSec... getting binary from' $AQUA_SCANNERCLI_URL \
+    && wget $AQUA_SCANNERCLI_URL -O aquasec \
+    && mv aquasec /usr/local/bin \
+    && chmod +rwx /usr/local/bin/aquasec \
+    && echo 'AquaSec CLI version:' \
+    && aquasec version \
+    && echo 'Aqua installation completed!'; \
```

Comment:
align with the "Installing...." message, so 'AquaSec installation completed'.

Reply:
Completely agree, I will change it